tools/scripts/fuzz_minimize_all.sh

```diff
@@ -1,20 +1,40 @@
 #!/bin/bash
 
-set -exuo pipefail
+set -euo pipefail
+cd "$(dirname "$0")"
+cd "../.."
+# we're now at the root of the repo.
 
-bazel build //test/fuzz:fuzz_dash_e --config=fuzz -c opt
+if [ "$#" -lt 1 ]; then
+cat <<EOF
+usage:
+  $0 <target> [<options>]
 
-# shellcheck disable=SC2207
-input_src=(
-    $(find 'fuzz_crashers/original' -name "crash-*" | sort)
-)
+example target:
+  fuzz_dash_e
+  fuzz_doc_symbols
+  fuzz_hover
 
-COMMAND_FILE=$(mktemp)
+example options:
+  --stress-incremental-resolver
+EOF
+exit 1
+fi
 
-for this_src in "${input_src[@]}" DUMMY; do
-    if [[ ! "$this_src" == DUMMY ]]; then
-        echo "./tools/scripts/fuzz_minimize_crash.sh \"$this_src\" 2>/dev/null" >> "$COMMAND_FILE"
-    fi
+target="$1"
+shift
+
+echo "building $target"
+bazel build "//test/fuzz:$target" --config=fuzz -c opt
+
+echo "making command file"
+cmds="$(mktemp)"
+for f in fuzz_crashers/original/crash-*; do
+  # this breaks if any of the extra args in $@ have spaces, for instance if this script was called like this:
+  # $0 fuzz_dash_e --extra-opt 'foo bar'
+  echo "tools/scripts/fuzz_minimize_crash.sh $f $* 2>/dev/null" >>"$cmds"
```

Comment:
Are you forgetting to pass `target` to `fuzz_minimize_crash`?

Reply:
I am indeed.